docs/apm/troubleshooting.asciidoc

```diff
@@ -93,13 +93,19 @@ Instead, we should strip away the unique information and group our transactions
 In this case, that means naming all blog transactions, `/blog`, and all documentation transactions, `/guide`.
 
 If you feel like you'd be losing valuable information by following this naming convention, don't fret!
-You can always add additional metadata to your transactions using {apm-guide-ref-v}/metadata.html#labels-fields[labels] (indexed) or
-{apm-guide-ref-v}/metadata.html#custom-fields[custom context] (non-indexed).
+You can always add additional metadata to your transactions using {apm-guide-ref}/metadata.html#labels-fields[labels] (indexed) or
+{apm-guide-ref}/metadata.html#custom-fields[custom context] (non-indexed).
 
 After ensuring you've correctly named your transactions,
-you might still see an error in the APM app related to too many transaction names.
-If this is the case, you can increase the default number of transaction groups displayed in the APM app by configuring
-<<apm-settings-kb,`xpack.apm.ui.transactionGroupBucketSize`>>.
+you might still see errors in the APM app related to transaction group limit reached:
+
+`The number of transaction groups has been reached. Current APM server capacity for handling unique transaction groups has been reached. There are at least {numberOfMissingTransactionGroups} transactions missing in this list. Please decrease the number of transaction groups in your service or increase the memory allocated to APM server.`
```

Comment:
`{numberOfMissingTransactionGroups}` won't resolve in the docs

Reply:
@bmorelli25 I believe the intention behind the `{numberOfMissingTransactionGroups}` is it's simply acting as a placeholder as the exact number is unknown.

Reply:
yes, it's a variable in the UI. e.g `5 transactions`
How can I display that here in the docs?

Reply:
The text in the docs should either say: "There are at least 5 transactions missing in this list" or "There are at least X transactions missing in this list". No reason to put the placeholder text that we use internally.

Reply:
Thanks, I'll go for the `X`